tests/unit/dataactvalidator/test_c14_award_financial.py

```diff
@@ -0,0 +1,33 @@
+from tests.unit.dataactcore.factories.staging import AwardFinancialFactory
+from tests.unit.dataactvalidator.utils import number_of_errors, query_columns
+
+_FILE = 'c14_award_financial'
+
+def test_column_headers(database):
+    expected_subset = {'row_number', "fain", "uri", "piid"}
+    actual = set(query_columns(_FILE, database))
+    assert (actual & expected_subset) == expected_subset
+
+
+def test_success(database):
+    """ Test cases with different combinations of fain, uri, and piid """
+
+    # Test with only one present
+    award_fin_fain = AwardFinancialFactory(uri = None, piid = None)
+    award_fin_uri = AwardFinancialFactory(fain = None, piid = None)
+    award_fin_piid = AwardFinancialFactory(fain = None, uri = None)
+    # Test with all three
+    award_fin = AwardFinancialFactory()
```

Comment:
It does not fail this rule, which just requires at least one of the three to be present.  Revisions to C8, C9, and this rule cover the possible combinations (fain+uri will be a warning, piid with either fain or uri will be an error).  I created story DB-1244 to update these three rules to include the revisions made in the Validation_Rules_For_Broker sheet.